### PR TITLE
gzip setting has been removed from plone.app.caching

### PR DESCRIPTION
**Pull request by tkimnguyen**
for https://github.com/plone/documentation/issues/430

### CODE REVIEW

**Comment by svx** [comment]
In:
 /manage/deploying/front-end/nginx.rst

I would remove:
- http://httpd.apache.org/docs/2.2/mod/mod_deflate.html

You have this link already in the apache docs and having this under nginx could be confusing

**Comment by tkimnguyen** [comment]
Thanks for catching that - fixed.

File: manage/troubleshooting/buildout.rst [hunk 1]
```diff
@@ -528,3 +528,15 @@ Buildout download cache is corrupted. Run ``bin/buildout -vvv`` for more info. T
...
+============================================
+
+...using the Plone 5.0 unified installer on Mac OS X 10.11.1 El Capitan
+
+As per http://stackoverflow.com/questions/19548011/cannot-install-lxml-on-mac-os-x-10-9 you should update your Xcode command line tools::
+
+    xcode-select --install
+
+then re-run ``bin/buildout``. 
+
```
Comment (gforcada) [inline manage/troubleshooting/buildout.rst:542]: This chunk seems totally unrelated to gzip, no idea if it's on purpose or by accident, just asking...

**Comment by svx** [comment]
@gforcada I guess so too, and this makes the title of this pr a bit confusing :)

On a further note I would change the syntax:

"...using"

should be replaced with:

```
.. note::
```

or something, so that it looks a bit better :) and instead of the full link the usage of proper rst formatting would be better :) 

Example:
External hyperlinks, like `Python <http://www.python.org/>`_.

**Comment by tkimnguyen** [comment]
Ah yeah sorry I made the PR originally as per the subject line but other subsequent docs changes I was committing got automatically lumped in with this PR. How could I keep them separate?

**Comment by tkimnguyen** [comment]
Going to close this PR and make a new one that doesn't include the commit about lxml.

**Comment by tkimnguyen** [comment]
https://github.com/plone/documentation/pull/439 and https://github.com/plone/documentation/pull/440
